### ISSUE TITLE
Create controllers and basic CRUD for youth savings years

### ISSUE THREAD

**Issue by elasticspoon** [rubyforgood/casa#5753]
## What type(s) of User Does This Feature Affect?

 - supervisors
 - admins
 - volunteers

## Description

Part of #3027
Blocked by #5752

Youth 14+ in foster care in MD are supposed to receive money deposited into a “Foster Youth Savings” account each year. 

We have created tables and models to track the deposits towards a case each fiscal year. Now we need to be able to actually create and modify those deposits. That means building out the CRUD and controllers needed for this functionality and the super basic views to display it. 

Some things to consider:

- how should we route this feature? Should this be a nested resource? 
- Don't worry about linking up these initial views, that will come later
- Consider what actions we need. Should we have more actions than just basic CRUD? Do we need actions for creation or can that be automatic?
- Do we need more than just an index page? Can we just do all the editing with turbo frames?

### Criteria for Completion

- [ ] please tell me you came up with something better than youth savings year
- [ ] Build out a working show page view. it should display complete and incomplete items for the year and a monetary value (maybe this can count down from a maximum instead of counting up to encourage getting that 💵 )
- [ ] Build functionality to edit values for a savings year
- [ ] Build out an index that displays all the savings years for a particular case

## How to Access the QA Site

_Login Details:_
[Link to QA site](https://casa-qa.herokuapp.com/)

Login Emails:
- volunteer1@example.com view site as a volunteer
- supervisor1@example.com view site as a supervisor
- casa_admin1@example.com view site as an admin
- all_casa_admin1@example.com view site as an all casa admin
  - go to `/all_casa_admins/sign_in`

password for all users: 12345678

### Questions? Join Slack!

We highly recommend that you join us in [slack](https://join.slack.com/t/rubyforgood/shared_invite/zt-21pyz2ab8-H6JgQfGGI0Ab6MfNOZRIQA) #casa channel to ask questions quickly. And [discord](https://discord.gg/qJcw2RZH8Q) for office hours (currently Tuesday 5-7pm Pacific), stakeholder news, and upcoming new issues.

**Comment by compwron** [comment]
Closing for inactivity — this issue has been open for a long time with no recent activity, and is most likely no longer relevant to the current state of the project (or has been overtaken by events).

If this still matters to you or has resurfaced, please feel free to:
- **Reopen** with a quick comment on why it's still needed, or
- **Open a fresh issue** describing the current behavior and what you'd like to see — much easier than picking up multi-year-old context.

Thanks for understanding 💜
